server/models/Request.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+    required: true
+  },
+  sitter_id: {
+    type: String,
+    required: true
+  },
+  start: {
+    type: Date,
+    required: true
+  },
+  end: {
+    type: Date,
+    required: true
+  },
+  accepted: {
```

Comment:
Maybe it would be better to use a single Boolean value than use two (accepted and declined) to indicate whether the sitter has accepted unless it was specified in docs given :-)

Reply:
Yes. That would be better, I will make that change immediately

Reply:
I checked the docs and it was specified. and I think the reason is that a request that has not been accepted has not necessarily been declined, which explains the default value of `false` for both `accepted` and `declined`.  i.e upon creating the request, the request has neither been accepted nor has it been declined. so with that in mind, I'll make a few adjustments. if you have any thoughts please let me know :-)

Reply:
Hey @BerryGIT-ME, you've understood it correctly :)

If we only had one field, the status of the booking would either be accepted/declined, we won't a way of showing 'pending' requests which have not been acted upon by the user.